### DISCUSSION TITLE
v23 bug 在打开全屏启动器的时候win+D并不会自动关闭启动器

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4423] [Troubleshooting | 故障排查]
期望的行为：按win+D显示桌面的时候启动器自动关闭，再次按win+D的时候还原窗口时不自动出现

现在的行为：启动器像普通窗口一样被win+D放置于后台，直接导致在win+D之后首次按win无法唤出启动器（因为被认为是操作关闭启动器）
转:神末shenmo

**Comment by BLumia** [comment]
转到 https://github.com/linuxdeepin/developer-center/issues/4432 跟进吧。

**Reply by Linux-qitong** [reply]
怎么转啊，不怎么会。

**Reply by BLumia** [reply]
啊？我的意思是这个问题的后续你可以直接在那个 issue 里跟进，不再在这个 discussion 里跟进了。你不需要做额外操作的（但如果你有兴趣跟进这个问题的修复情况的话，可以到那个 issue 里点一下右侧最下面的 subscribe 来订阅后续进度）。
